src/components/Hero.js

```diff
@@ -2,24 +2,26 @@ import React, { useState } from "react";
 import styled from "styled-components";
 import { IoMdCloseCircle } from "react-icons/io";
 
-const close = (isVisible, setIsVisible) => {
-  setIsVisible(!isVisible);
-}
-
 export default function Hero(props) {
-  const [isVisible, setIsVisible] = useState(true)
+  const verifactHeroClosed = JSON.parse(localStorage.getItem('verifactHeroClosed')) ? true : false;
+  const [isClosed, setIsClosed] = useState(verifactHeroClosed)
   const welcomeTitle = "Your home for verifying credible news"
   const welcomeContent = "Post a question to our community of news sleuths to get answers and new perspectives about the news your reading"
 
-  return <>{isVisible ? (<Wrapper>
+  const close = () => {
+    localStorage.setItem('verifactHeroClosed', 'true');
```

Comment:
I still think this should be abstracted so that it can be used elsewhere instead of calling JSON.parse directly everywhere. Then you don't have to think if you are correctly serializing and parsing. You might add this to `src/utils`. Here is an example utility that could accomplish this:

```
export function setLocalData (key, value) {
  if (!value) return
  window.localStorage.setItem(
    key,
    typeof value == 'string' ? value : JSON.stringify(value)
  )
}

export function getLocalData (key) {
  const item = window.localStorage.getItem(key)
  return item ? JSON.parse(item) : null
}

export function removeLocalData (key) {
  window.localStorage.removeItem(key)
}

```

Reply:
Changed in `f980c26`